Exercises/operators-and-type-conversions/operators-and-type-conversions.js

```diff
@@ -0,0 +1,21 @@
+// Task #1
+a = 10;
```

Comment:
don't do variables without `let/cons` they will appear in global object and will be available everywhere